discovery_engine_core/web-api/src/migrations/20220926145400_document.sql

```diff
@@ -0,0 +1,24 @@
+--  Copyright 2022 Xayn AG
+--
+--  This program is free software: you can redistribute it and/or modify
+--  it under the terms of the GNU Affero General Public License as
+--  published by the Free Software Foundation, version 3.
+--
+--  This program is distributed in the hope that it will be useful,
+--  but WITHOUT ANY WARRANTY; without even the implied warranty of
+--  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--  GNU Affero General Public License for more details.
+--
+--  You should have received a copy of the GNU Affero General Public License
+--  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+CREATE TABLE IF NOT EXISTS document (
+    doc_id TEXT NOT NULL,
+    user_id TEXT NOT NULL,
```

Comment:
the ticket states
> We keep an association between the user and the document on postgres.
does it mean that we wanted to add a `user` table and FK to other tables? @acrrd

Reply:
with don't have a table of user to reference

Reply:
not yet

Reply:
still, we cannot create an FK to a table that does not exist :)
for the future, I don't know. Maybe it will pop up.